src/vs/gitpod/node/gitpodInsightsAppender.ts

```diff
@@ -191,7 +193,7 @@ function mapTelemetryData(eventName: string, data: any): RemoteTrackMessage | un
 			}
 			filterEvent.add(eventName);
```

Comment:
I don't think it is good guard. You need to add both `editorOpened` and `fileGet`.

Maybe it is better to have explicit flags like: `readAccessTracked` and `writeAccessTracked`. If they are already true then return undefined.

Reply:
done